src/components/send/SendAssetFormCollectible.js

```diff
@@ -108,19 +108,18 @@ const SendAssetFormCollectible = enhance(
         )}
       </Column>
       <View
-        width="100%"
-        marginBottom={0}
         height={210}
         justifyContent="flex-end"
+        marginBottom={0}
+        width="100%"
       >
         <ColumnWithMargins
-          flex={0}
           margin={deviceUtils.dimensions.height < 812 ? 15.5 : 25}
-          style={{ zIndex: 3 }}
-          width="100%"
           marginBottom={29}
-          paddingHorizontal={15}
           paddingBottom={15}
+          paddingHorizontal={15}
+          style={{ zIndex: 3 }}
```

Comment:
We should move this out

Reply:
ya tbh this whole file needs to be rewritten but i dont think we should do it right now